README.md

```diff
@@ -90,3 +90,24 @@ does not support development on Windows or z/OS.
 [discord-link]: https://discord.gg/hHaz2gGX
 [gpl-v3-badge]: https://img.shields.io/badge/License-GPLv3-blue.svg?style=flat
 [gpl-v3-link]: https://www.gnu.org/licenses/gpl-3.0
+
+### Docker
+
+This project includes a `Dockerfile` which creates a build container based on `ubuntu-22.04` on the `arm64` platform.  This allows development on `MacOS` to proceed without worrying about the complex dependencies, and without having to pipe data from the internt directly into a shell.
+
+#### Build the build container
+```
+docker build -t sbtc-build .
+```
+
+#### Run the build container
+```
+docker run -it -v `pwd`:/build sbtc-build
+```
+
+#### Build using the build container
+```
+cd /sbtc
+make install
+make build
```

Comment:
For my own education, can you elaborate on why this works? The Makefile hasn't received any edits so I don't understand how this would do what it does.